workers/ohsome_quality_analyst/api/api.py

```diff
@@ -119,8 +130,16 @@ def empty_api_response() -> dict:
     return {
         "apiVersion": __version__,
         "attribution": {
-            "text": "© OpenStreetMap contributors",
-            "url": "https://ohsome.org/copyrights",
+            "text": (
+                "© OpenStreetMap contributors;"
+                "© European Union, 1995-2022"
+                "Global Human Settlement Layer Data;"
```

Comment:
It should be one or the other.
See:
https://ec.europa.eu/info/legal-notice_en
and
https://ghsl.jrc.ec.europa.eu/faq.php

Reply:
@joker234 Your opinion on that is welcome.

Reply:
That's really not pretty:
```
  "attribution": {
    "text": "© OpenStreetMap contributors;© European Union, 1995-2022Global Human Settlement Layer Data;Earth Observation Group Nighttime Light Data.",
    "url": "https://github.com/GIScience/ohsome-quality-analyst/blob/main/data/LICENSES.md"
  },
```
I'll leave a suggestion separately as this comment does not cover the necessary lines.